Module/randomize.py

```diff
@@ -198,11 +237,10 @@ def setBonusStats(self):
                     random.shuffle(statsList)
                 location.setStat(statsList.pop())
 
-        print(len(statsList))
-
-    def validateSeed(self, startingInventory):
+    def validateSeed(self, startingInventory, reverseRando=False):
```

Comment:
lets change reverseRando to be restricted_config or something and just be a dict of restricted_treasures and restricted_bonuses.

Or if you prefer it could be a function that is passed in for the type of validation done.

This would make it easier to add additional modes of validation (Maybe for other mods in the future, such as Xaddgx's Aqua mod, or just for different modes of play)

Reply:
Also thinking it would probably make sense for validateSeed to be moved into its own class, and it should take a seed as an argument to validate. I thinking having that logic separate will make testing easier.

If this is too difficult to do the way the randomizer is structured, let's look at ways to make that better with Tom.

Reply:
Then the last piece of this would be writing a few tests just so we can have some confidence that valid seeds are marked valid and invalid seeds are marked invalid.

I can write a starting set of these once the above two things are complete.

Reply:
Agreed on all these points. I'll see how feasible decoupling the validateSeed function is, and do so if possible. The restricted_config was obvious and I wish I did that earlier.